docs/manuals/core/config/scenario-config.md

```diff
@@ -15,77 +17,116 @@ A scenario configuration is necessary to instantiate a [Scenario](../concepts/sc
+
+scenario_cfg = Config.configure_scenario("multiply_scenario",
+                                         [add_task_cfg_1, add_task_cfg_2, multiply_task_cfg_1, multiply_task_cfg_2],
+                                         sequences={"add_sequence": [add_task_cfg_1, add_task_cfg_2,],
+                                                    "multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})
+```
 
-When the scenario configuration contains only one single pipeline configuration, we can also create the
-`ScenarioConfig^` from the task configurations directly.
+We can also add sequences to your scenario configuration after you have created your scenario configuration.
 
 ```python linenums="1"
 from taipy import Config
 
-scenario_cfg = Config.configure_scenario_from_tasks("multiply_scenario",
-                                                    [task_cfg])
+scenario_cfg = Config.configure_scenario("multiply_scenario",
+                                         [add_task_cfg_1, add_task_cfg_2, multiply_task_cfg_1, multiply_task_cfg_2],)
+
+scenario_cfg.add_sequences({"add_sequence": [add_task_cfg_1, add_task_cfg_2,],
+                            "multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})
+
+# Or we can add them separately as well
+scenario_cfg.add_sequences({"add_sequence": [add_task_cfg_1, add_task_cfg_2,],})
+scenario_cfg.add_sequences({"multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})
+
 ```
 
-Behind the scenes, a pipeline configuration is created. Its id will be the scenario configuration id with the
-`_pipeline` postfix (`multiply_scenario_pipeline` in the example).
+We can also remove sequences after providing them in your scenario configuration.
+
+```python linenums="1"
+from taipy import Config
 
-!!! Note
+scenario_cfg = Config.configure_scenario("multiply_scenario",
+                                         [add_task_cfg_1, add_task_cfg_2, multiply_task_cfg_1, multiply_task_cfg_2],
+                                         sequences={"add_sequence": [add_task_cfg_1, add_task_cfg_2,],
+                                                    "multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})
 
-    Note that the pipeline id can be configured as an optional parameter as follows:
+scenario_cfg.remove_sequences(["add_sequence", "multiply_sequence"])
 
-    ```python linenums="1"
-    from taipy import Config
+# Or we can remove a single sequence from a ScenarioConfig by passing only a sequence name to
+# ScenarioConfig.remove_sequences
+scenario_cfg.remove_sequences("add_sequence")
 
-    scenario_cfg = Config.configure_scenario_from_tasks(
-        "multiply_scenario",
-        [task_cfg],
-        pipeline_id="multiply_pipeline")
-    ```
+```
+In the small examples above, we create a scenario configuration `ScenarioConfig^` from task configurations and
+defined sequences of that scenario configuration. You can refer back to the sequence concept in
```

Comment:
"defined sequences of that scenario configuration" -> "we define sequences for that scenario configuration"